src/components/Typography/docs.md

```diff
@@ -46,63 +46,227 @@ import {colors: {text}, fontStyles: {serifRegular21}} from '@project-r/styleguid
+    One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed into a horrible vermin. He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections.
+  </Editorial.P>
+  <Editorial.Subhead>The quick brown fox jumps over the lazy dog</Editorial.Subhead>
+  <Editorial.P>
+    The bedding was hardly able to cover it and seemed ready to slide off any moment. His many legs, pitifully thin compared with the size of the rest of him, waved about helplessly as he looked. "What's happened to me?" he thought. It wasn't a dream.
+  </Editorial.P>
+  <Editorial.P>
+     His room, a proper human room although a little too small, lay peacefully between its four familiar walls. One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed into a horrible vermin.
+  </Editorial.P>
+</article>
+```
+
+### Paragraphs and Questions
+
+```react|responsive
+<article>
+  <Editorial.Question>
+    What happened to Mr. Samsa?
+  </Editorial.Question>
+  <Editorial.Answer>
+    <Editorial.AnswerBy>Franz Kafka</Editorial.AnswerBy> One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed into a horrible vermin. He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections.
+  </Editorial.Answer>
+  <Editorial.Question>
+    What happened next?
+  </Editorial.Question>
+  <Editorial.Answer>
+    The bedding was hardly able to cover it and seemed ready to slide off any moment. His many legs, pitifully thin compared with the size of the rest of him, waved about helplessly as he looked. "What's happened to me?" he thought. It wasn't a dream.
+  </Editorial.Answer>
+</article>
+```
+
+### QuoteText and Cite
+
+These are used by the [PullQuote](/pullquote) component.
+
+```react
+<blockquote>
+  <Editorial.QuoteText>
+    Ich bin sicher, eine kleine Rebellion hie und da ist eine gute Sache; sie ist in der Politik so notwendig, um die Dinge zu klären, wie ein Sturm für das Wetter.
+  </Editorial.QuoteText>
+  <Editorial.Cite>
+  Thomas Jefferson
+  </Editorial.Cite>
+</blockquote>
+```
+
+### BoxTitle and BoxText
+
+These are used by the [InfoBox](/infobox) component.
+
+```react
+<div>
+  <Editorial.BoxTitle>
+    This is a box title
+  </Editorial.BoxTitle>
+  <Editorial.BoxText>
+    One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed into a horrible vermin. He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections. The bedding was hardly able to cover it and seemed ready to slide off any moment.
+  </Editorial.BoxText>
+</div>
+```
+
+### Caption and Byline
+
+These are used by the [Figure](/figure) component.
+
+```react
+<figure>
```

Comment:
Probably should be `<Figure>` and `<Image>`

Reply:
Removed.